training/model_evaluation/components/pipeline_component/spec.yaml

```diff
@@ -99,7 +101,7 @@ jobs:
   
   compute_metrics:
     type: command
-    component: azureml://registries/azureml/components/compute_metrics/versions/0.0.5
+    component: azureml://registries/azureml-preview/components/compute_metrics/versions/0.0.6
```

Comment:
azureml://registries/azureml/components/compute_metrics/versions/0.0.6 * 
preview should be removed